src/order/OrderAPI.ts

```diff
@@ -139,12 +145,12 @@ export class OrderAPI {
    * orders are returned. As soon as an order is no longer open and settled, it will no longer appear
    * in the default request.
    *
-   * @param pagination - Pagination field
+   * @param query - Available query parameters (Pagination, Product ID and/or Order Status)
    * @see https://docs.pro.coinbase.com/#list-orders
    */
-  async getOpenOrders(pagination?: Pagination): Promise<PaginatedData<Order>> {
+  async getOpenOrders(query?: OrderListQueryParams): Promise<PaginatedData<Order>> {
```

Comment:
Extending the functionality of this method makes sense, as I suspect people are already using it in this manner. However, the name of the method still implies a particular order, namely those which are "open". A general "getOrders" method is merited to supplement the current offering, IMHO.

Reply:
When you give it a "status" of "all", will it then also return "closed" orders? Because if not, then we can still use the method `getOpenOrders`.

Reply:
As I understand it, "open" is a specific _state_ an order can be in. An order cannot be both "open" and "pending". So it seems odd to call `getOpenOrders({ status: 'pending' })` since the method-name and parameters are inconsistent. As the documentation states, "Passing all returns orders of all statuses." A status of "all" is all-inclusive 🙂